### ISSUE TITLE
When posting kotter to a channel - tag @channel instead of @slack channel ID

### ISSUE THREAD

**Issue by kofkne** [F3Nation-Community/weaselbot#9]

**Comment by kofkne** [comment]
requested by Huggies from F3 Phoenix

**Comment by jasheldo** [comment]
Hi @kofkne . Could you give me an example of what you're looking for? Tagging a channel uses the octothorp (#). Tagging an individual uses ampersand (@). I just want to make sure I understand your ask correctly.

**Comment by kofkne** [comment]
![Screenshot_20240310-105009](https://github.com/F3Nation-Community/weaselbot/assets/83671789/e241943b-3ace-435f-a1cc-108433177f9e)

@jasheldo check the screenshot for what was posted to our kotter channel. 

All we see is "Hello, @...!" 

I would prefer it to tag @channel

**Comment by jasheldo** [comment]
What region are. you with brother? I _think_ I see the issue.

**Comment by kofkne** [comment]
Phoenix.

**Comment by jasheldo** [comment]
@kofkne - I'll be taking over the reins for Weaselbot to give @evanpetzoldt some time to focus on other projects. Part of that includes a rewrite of some of the logic. Look at the current logic, NA's aren't dropped at that stage so the Kotter Report gets sent to a ghost user. In the new logic, NA's are dropped so there would be no ghost user message that would be sent.

That begs the question... when the new code goes into production, is there a need, when there are no actual site q users in the table, to send one message out to `@channel`? That may not be a bad way to go but It's impactful to many regions so I want to make sure @evanpetzoldt is okay with this.

**Comment by kofkne** [comment]
So when I read the read me, I could designate site q or a channel to have the report posted to. I went with the channel. 

I set up a private channel and only invited selected pax to help work the kotter report. 

I would think the logic will be that if no site Q and it is posting to a channel to use the @channel to notify everybody in the channel. 

If it's not possible, totally okay. It was just a suggestion/request. I would think other regions that have their report going to a channel and not specific people, probably wouldn't mind getting the @channel tag so everyone is notified in the channel they selected

It would be easier to talk more in slack, I'm in the paxminer group under Huggies (Phoenix, AZ)

**Comment by evanpetzoldt** [comment]
So yeah, when I initially set up Weaselbot, I only set up the full Kotter reports to go to one PAX through the `default_siteq` field (using their `user_id`, but then I realized that I could put a `channel_id` in there instead and it would still work... I just never bothered to change the code to handle the way it greets the channel instead of the user. User_ids start with U, so we could just have logic that either starts with f"Hi, <@{default_siteq}>" if it starts with a U and either "Hi, <#{default_siteq>" or "Hi, @channel" if it starts with something else.

@jasheldo , the just so you know the logic should be: send the full Kotter list to the region's `default_siteq`, and then send AO-filtered lists to siteqs listed on thier `aos` table if they have them.
